packages/viewer/src/lib/types/entities.ts

```diff
@@ -26,7 +26,7 @@ export enum EntityType {
   IMPRECISE_EDGE = 0xa0, // == 160
 
   /**
-   * A value that represents the presence of an surface without BREP.
+   * A value that represents the presence of a surface without BREP.
```

Comment:
Nice catch! Honestly we could probably use a pass through the SDK to determine if there are other cases of typos - it doesn't look great in documentation and these seem keep popping up. Definitely not something to include here of course, just a thought. Thanks for fixing these!